src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -226,6 +246,22 @@ public Index checkIfRoomPresent(Integer roomNumber) {
         return index;
     }
 
+    @Override
+    public void updateRoomListWhenPatientsChanges(Patient patientToEdit, Patient editedPatient) {
+        ObservableList<Room> roomObservableList = this.getModifiableRoomList().getRoomObservableList();
```

Comment:
Would you like to switch to `getRoomList()` here? We will eventually remove `getModifiableRoomList`.

Reply:
mayeb i will use `this.roomList.getRoom...`, because `getRoomList` returns the unmodifiable version

Reply:
Oh yeah, this wouldn't work if you use the unmodifiable version. Feels like using the modifiable version is a hack though hahaha.

Reply:
yup. i dont really feel its a hack tho, cuz the `ModelManager` is supposed to interact with the roomlist directly. We should make it so that other classes cannot interact with the roomlist directly and remove the `getModifiableRoomList` though

Reply:
Actually, we probably should have another class similar to `PatientRecord` to interact with `RoomList`.

Reply:
that is true, we can look into this for next week